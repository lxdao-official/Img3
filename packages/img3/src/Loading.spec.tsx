import { render, screen } from '@testing-library/react';
import React from 'react';
import { Loading } from './Loading';

test('renders Loading', () => {
  const { asFragment } = render(<Loading />);
  expect(asFragment()).toMatchSnapshot();
});
