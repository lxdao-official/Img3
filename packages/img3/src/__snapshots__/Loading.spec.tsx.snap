// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Loading 1`] = `
<DocumentFragment>
  <div
    class="sc-bcXHqe jwdRIs"
    color="rgba(0,0,0,0.9)"
    size="40"
  >
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
  </div>
</DocumentFragment>
`;
